src/instrumentation/queue.ts

```diff
@@ -135,6 +136,7 @@ export function executeQueueHandler(queueFn: QueueHandler, [batch, env, ctx]: Qu
 	const tracer = trace.getTracer('queueHandler')
 	const options: SpanOptions = {
 		attributes: {
+			[SemanticAttributes.FAAS_TRIGGER]: 'pubsub',
```

Comment:
I agree with the comments that we should probably make this either `queue` or `cf.queue` or something so that we can distinguish when PubSub gets a trigger.